code/stable/glassbr/src/cpp/Calculations.cpp

```diff
@@ -18,52 +15,55 @@ using std::vector;
 using std::ifstream;
 using std::ofstream;
 
+#include "InputParameters.hpp"
+#include "Interpolation.hpp"
+
 bool func_is_safePb(InputParameters &inParams, double P_b) {
-    return P_b < inParams.P_btol;
+    return (P_b < inParams.P_btol);
```

Comment:
Do you know why there are extra `()` here? [Not a big deal, but it would be nice to not have them. Probably future work.]

Reply:
Please file an issue. It's a fun feature to implement, but it is 'work'.

Reply:
#1513 created.